src/components/Header/HeaderControls.tsx

```diff
@@ -88,6 +88,7 @@ export function HeaderControls({ setIsCodeFlyoutVisible }: IHeaderControls) {
         <ControlButton
           aria-label="Toggle the script recorder between recording and paused"
           color="primary"
+          isDisabled={isTestInProgress}
```

Comment:
FYI @justinkambic I think this was removed while rebasing the branch, added this before merging to the main branch